packages/provider/src/speech-model/v1/speech-model-v1.ts

```diff
@@ -0,0 +1,91 @@
+import { JSONValue } from '../../json-value';
+import { SpeechModelV1CallOptions } from './speech-model-v1-call-options';
+import { SpeechModelV1CallWarning } from './speech-model-v1-call-warning';
+
+/**
+ * Speech model specification version 1.
+ */
+export type SpeechModelV1 = {
+  /**
+   * The speech model must specify which speech model interface
+   * version it implements. This will allow us to evolve the speech
+   * model interface and retain backwards compatibility. The different
+   * implementation versions can be handled as a discriminated union
+   * on our side.
+   */
+  readonly specificationVersion: 'v1';
+
+  /**
+   * Name of the provider for logging purposes.
+   */
+  readonly provider: string;
+
+  /**
+   * Provider-specific model ID for logging purposes.
+   */
+  readonly modelId: string;
+
+  /**
+   * Generates speech audio from text.
+   */
+  doGenerate(options: SpeechModelV1CallOptions): PromiseLike<{
+    /**
+     * The audio data as a binary buffer.
+     */
+    audioData: ArrayBuffer;
+
+    /**
+     * The MIME type of the audio data.
+     * For example: 'audio/mp3', 'audio/wav', etc.
+     */
+    contentType: string;
+
+    /**
+     * Warnings for the call, e.g. unsupported settings.
+     */
+    warnings: Array<SpeechModelV1CallWarning>;
+
+    /**
+     * Optional request information for telemetry and debugging purposes.
+     */
+    request?: {
+      /**
+       * Raw request HTTP body that was sent to the provider API as a string (JSON should be stringified).
+       * Non-HTTP(s) providers should not set this.
+       */
+      body?: string;
```

Comment:
please change to `body?: unknown` to be aligned with recent changes on v5 branch (see also example docs text in v5 branch language model v2)